conf/ds-present-kb-collections.yaml

```diff
@@ -90,6 +90,7 @@ config:
                     injections:
                       #- imageserver: ${path:config.imageservers[default=true].url}
                       - streamingserver: "here_should_be_a_wowza_url"
+                      - old_imageserver: ${path:config.imageservers.oldimageserver.url}
```

Comment:
Sorry, but naming it `oldimageserver` in one place and `old_imageserver` in another place earns you another round in the review process!

Reply:
Considering this a second time: No, you are right! While the YAML has the convention of concatenating lowercased words, the XSLT uses underscore. So you did the right thing. My apologies!